app/cinema/hall.py

```diff
@@ -0,0 +1,19 @@
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self: 'CinemaHall', number: int) -> None:

```

Comment:
You should not provide type hint for `self`
```suggestion
    def __init__(self, number: int) -> None:

```

Reply:
flake8 write that i should provide it